config/analytics_blocklist.yml

```diff
@@ -1,5 +1,5 @@
 ---
-shared:
+:shared:
```

Comment:
Is this change significant?

Reply:
A lot of these changes were done automatically. The prefixed `:` seems to be consistent with the rest of the file though.